### PR TITLE
bump substrate@b9cafb

### PR DESCRIPTION
**Pull request by clangenb**
Last commit before a breaking change where AccountId does no longer derive default.

### CODE REVIEW

**Review by haerdib — COMMENTED** [review]
Happens if you increase the rust toolchain drastically 😄 

Regarding AccountId - before finding a substitution, such as encoding, we might consider switching to OptionQueries, these don't enforce default implementations. But I haven't looked into it yet regarding the substrate-api-client.

**Comment by clangenb** [comment]
Yes, I think we will make that switch to the option queries. I also don't know if it requires changes to the api-client at all. I just wanted to give some context, why I did not pick the newest one.

Afaik however, this is only supported in the new frame v3 syntax. We don't have this syntax yet in any encointer-pallet...

File: Cargo.lock [hunk 1]
```diff
@@ -3426,7 +3391,7 @@ version = "1.6.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "1f559b464de2e2bdabcac6a210d12e9b5a5973c251e102c44c585c71d51bd78e"
 dependencies = [
- "cfg-if 1.0.0",
+ "cfg-if 0.1.10",
```
Comment (haerdib) [inline Cargo.lock:3394]: Unsure if that's a good change. Might be worth to rerun cargo update

File: node-api/src/error.rs [hunk 2]
```diff
@@ -95,6 +95,9 @@ pub enum RuntimeError {
     /// At least one consumer is remaining so the account cannot be destroyed.
     #[error("At least one consumer is remaining so the account cannot be destroyed.")]
     ConsumerRemaining,
+    /// There are too many consumers so the account cannot be created.
```
Comment (haerdib) [inline node-api/src/error.rs:98]: Seems to be quite redundant. Is it really necessary?

**Review by haerdib — APPROVED** [review]
LGTM. Minor comments, but feel free to merge if CI passes.

File: Cargo.lock (hunk 1 above)
Comment (clangenb) [reply Cargo.lock:3394]: This is very interesting, it is always this dependency, where this occurs. I have no idea why cargo does that.

File: node-api/src/error.rs (hunk 2 above)
Comment (clangenb) [reply node-api/src/error.rs:98]: I agree with you, I just took it from the original error in substrate.

**Pull request merged**